### PR TITLE
fix(schema): make item _id a string type

### PR DESCRIPTION
**Pull request by petrjasek**
instead of objectid which won't work for guid.

SD-4298

### CODE REVIEW

File: features/publish.feature [hunk 1]
```diff
@@ -5,13 +5,23 @@ Feature: Publish service
     	"""
     	[{
             "guid": "tag:example.com,0000:newsml_BRE9A605",
-            "type": "picture",
+            "body_html": "<p>Lorem ipsum</p>",
+            "byline": "john",
+            "description_text": "lorem ipsum",
+            "guid": "tag:example.com,0000:newsml_BRE9A605",
```
Comment (mugurrus) [inline features/publish.feature]: guid field is set twice.

**Comment by mugurrus** [comment]
items is a readonly resource and _id is removed from the response. What do you mean "send _id with push"?

**Comment by petrjasek** [comment]
this description was obsolete, updated it

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**